kubectl-pg/README.md

```diff
@@ -0,0 +1,64 @@
+# kubectl-pg
+
+### Installtion of kubectl pg plugin
+
+This project uses Go Modules for dependency management to build locally
+Install go and enable go modules ```export GO111MODULE=on```
```

Comment:
Explicitly enabling modules will be unnecessary for Go versions >= 1.13 (should be released next month) since this version will [enable modules by default](https://tip.golang.org/cmd/go/#hdr-Module_support)